packages/cli-tool/src/add-ons/ui-framework/index.ts

```diff
@@ -16,8 +17,9 @@ export const setUIFramework = async(
     cli.info('Configure Bootstrap...');
 
     await setupBootstrap(appName);
-  }
-  if (uiFramework === 'tailwind') {
-    cli.info('Tailwind is not available yet. Please configure it manually.');
+  } else if (uiFramework === 'tailwind') {
```

Comment:
How about converting to switch-case for further options then? Also, the name should be `tailwind-css`, end-users gonna understand either way but rather could be more preciously. What do you think

Reply:
Good idea! 

Tried in 39d1482 ;-)